### PR TITLE
brackets: drop

### LINKED ISSUE #59196: brackets needs update
## brackets is out of date
Current version of brackets in nixpkgs is `1.9` while the latest release is [1.13](https://github.com/adobe/brackets/releases/tag/release-1.13) as of Jun 18, 2018.

### LINKED ISSUE #90740: Vulnerability roundup 85: brackets-1.9: 1 advisory [9.8]
[search](https://search.nix.gsc.io/?q=brackets&i=fosho&repos=NixOS-nixpkgs), [files](https://github.com/NixOS/nixpkgs/search?utf8=%E2%9C%93&q=brackets+in%3Apath&type=Code)

* [ ] [CVE-2019-8255](https://nvd.nist.gov/vuln/detail/CVE-2019-8255) CVSSv3=9.8 (nixos-20.03, nixos-unstable)

Scanned versions: nixos-20.03: a84b797b28e; nixos-unstable: 22c98819ccd. May contain false positives.

### PR DESCRIPTION
**Pull request by dotlambda**
###### Motivation for this change
Hasn't been updated since 2017 and suffers from CVE-2019-8255.
closes https://github.com/NixOS/nixpkgs/issues/90740
closes https://github.com/NixOS/nixpkgs/issues/59196

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
The last release is from 6. April 2020. https://github.com/adobe/brackets/releases

File: pkgs/top-level/aliases.nix [hunk 1]
```diff
@@ -70,6 +70,7 @@ mapAliases ({
   bazaarTools = throw "bazaar has been deprecated by breezy."; # added 2020-04-19
   beegfs = throw "beegfs has been removed."; # added 2019-11-24
   bluezFull = bluez; # Added 2019-12-03
+  brackets = throw "brackets has been removed, it was unmaintained and had open vulnerabilities";
```
Comment (worldofpeace) [inline pkgs/top-level/aliases.nix:73]: can u add a date on this line?

**Comment by worldofpeace** [comment]
Yep, lets remove this.

**Review by worldofpeace — APPROVED** [review]

**Comment by worldofpeace** [comment]
@dotlambda Thanks :sparkles:

**Pull request merged**